### PR TITLE
fix (msw/2/imports): update package versions and tsconfig.json

### PR DESCRIPTION
**Pull request by amirabbas-gh**
#### 📚 Description

This PR addresses a critical error found in the `tsconfig.json` file and fixes several issues related to packages versions in package.json. The changes include:

Updating tsconfig.json with the correct configuration to resolve runtime errors.
Update `ts-morph`, `typescript`, `@vitest/coverage-v8` and `@codemod.com/codemod-utils` versions in package.json

#### 🧪 Test Plan

Updated the `tsconfig.json` file and verified it by running the project to ensure the error no longer occurs.
Update the `package.json`

#### 📄 Documentation to Update

- Regenerate thetsconfig.json file.
- Fix the package.json file.

### CODE REVIEW

File: msw/2/imports/tsconfig.json
Comment (mohebifar) [inline msw/2/imports/tsconfig.json:1]: same comment as the other pr

File: msw/2/imports/pnpm-lock.yaml
Comment (mohebifar) [inline msw/2/imports/pnpm-lock.yaml:1]: same comment as the other pr

File: msw/2/imports/.gitignore
Comment (mohebifar) [inline msw/2/imports/.gitignore:1]: you can move this .gitignore to the root. it'll ignore all `dist` and `node_modules` dirs in every subdir.

**Comment by amirabbas-gh** [comment]
@mohebifar i did it

**Review by mohebifar — APPROVED** [review]

**Pull request merged**